src/app/tests/suites/certification/Test_TC_CC.yaml

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2021 Project CHIP Authors
```

Comment:
The name of this file is odd.  I would have expected some testcase number bits in there....

Reply:
No test case number had been associated for the color loop so i left it blank. I named it the number 8.